### PR TITLE
Feature/climber basic commands

### LINKED ISSUE #24: Write basic commands for Climber
RunClimberCommand (Constructor 1)
- Should never end on its own.
- Should take a double (not a supplier). 
- Should `require` its subsystem.

RunClimberCommand (Constructor 2)
- Should never end on its own.
- Should take a double supplier. 
- Should `require` its subsystem.

StopClimberCommand
- Should never end on its own.
- Should `require` its subsystem.

### PR DESCRIPTION
**Pull request by citrus-melon**
Creates basic commands for the climber subsystem

- `RunClimber` to run the climber at a specific speed
- `StopClimber` to stop the climber

closes #24

### CODE REVIEW

File: src/main/java/frc/robot/commands/climber/RunClimberCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  /**
+   * Runs the {@link Climber} at the speed given by the supplier
+   * 
+   * @param climber The {@link Climber} to run
+   * @param speed Supplier for the speed to run at
+   */
+  public RunClimberCommand(Climber climber, Supplier<Double> speedSupplier) {
+    // Use addRequirements() here to declare subsystem dependencies.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/commands/climber/RunClimberCommand.java]: remove redundant autogenerated comments

File: src/main/java/frc/robot/commands/climber/RunClimberCommand.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    this.climber = climber;
+    this.speedSupplier = speedSupplier;
+    addRequirements(this.climber);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/commands/climber/RunClimberCommand.java]: remove redundant autogenerated comments

File: src/main/java/frc/robot/commands/climber/StopClimberCommand.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * Stops the {@link Climber} while active
+ */
+public class StopClimberCommand extends CommandBase {
+  private final Climber climber;
+
+  /**
+   * @param climber the {@link Climber} to stop
+   */
+  public StopClimberCommand(Climber climber) {
+    // Use addRequirements() here to declare subsystem dependencies.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/commands/climber/StopClimberCommand.java]: remove redundant autogenerated comments

File: src/main/java/frc/robot/commands/climber/StopClimberCommand.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    this.climber = climber;
+    addRequirements(this.climber);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/commands/climber/StopClimberCommand.java]: remove redundant autogenerated comments

File: src/main/java/frc/robot/commands/climber/RunClimberCommand.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  public RunClimberCommand(Climber climber, double speed) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this(climber, () -> speed);
+  }
+
+  /**
+   * Runs the {@link Climber} at the speed given by the supplier
+   * 
+   * @param climber The {@link Climber} to run
+   * @param speed Supplier for the speed to run at
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/commands/climber/RunClimberCommand.java]:
```suggestion
   * @param speedSupplier Supplier for the speed to run at
```

**Review by aidan-mundy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/climber/RunClimberCommand.java [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    climber.setSpeed(speedSupplier.get());
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/commands/climber/RunClimberCommand.java:47]: Remove this code; never used.

**Review by chit-sudeep — CHANGES_REQUESTED** [review]
Looked over all of the code and javadocs.

Implement Aidan's comments.

Look at the code I request you to remove.

File: src/main/java/frc/robot/commands/climber/RunClimberCommand.java (hunk 6 above)
Comment (citrus-melon) [reply src/main/java/frc/robot/commands/climber/RunClimberCommand.java:47]: I believe the code is used when the command is interrupted by something outside of itself.

**Review by chit-sudeep — DISMISSED** [review]
Accidental approval, ignore

**Review by aidan-mundy — DISMISSED** [review]

File: src/main/java/frc/robot/commands/climber/StopClimberCommand.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+import frc.robot.subsystems.Climber;
+
+/**
+ * Stops the {@link Climber} while active
+ */
+public class StopClimberCommand extends CommandBase {
+  private final Climber climber;
+
+  /**
+   * @param climber the {@link Climber} to stop
```
Comment (chit-sudeep) [inline src/main/java/frc/robot/commands/climber/StopClimberCommand.java]: Capitalize "the"

**Review by chit-sudeep — CHANGES_REQUESTED** [review]
Please review requested change.

**Review by chit-sudeep — APPROVED** [review]
Reviewed all code and javadocs.

Looks good!

**Review by aidan-mundy — APPROVED** [review]

**Pull request merged**